deny.toml

```diff
@@ -36,8 +36,16 @@ deny = [
     # https://github.com/briansmith/ring/issues/774
     { name = "ring" },
 ]
-skip = []
-skip-tree = []
+skip = [
```

Comment:
Not fond of this, a little surprised there isn't a way to allow path dependencies and only check their transitive dependencies.
This may all go away though if #125 collapses to opaque types as the common test logic would no longer be needed in `mc-sgx-tservice-types`

Reply:
gone now